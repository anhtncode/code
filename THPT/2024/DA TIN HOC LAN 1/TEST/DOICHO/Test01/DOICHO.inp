49
6 2 6 2 1 9 1 6 2 6 2 1 9 1 6 2 6 2 1 9 1 6 2 6 2 1 9 1 6 2 6 2 1 9 1 6 2 6 2 1 10 1 6 2 6 2 1 10 1
