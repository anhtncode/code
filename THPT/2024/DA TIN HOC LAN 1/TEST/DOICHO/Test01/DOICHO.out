6 2 6 2 10 9 1 6 2 6 2 1 9 1 6 2 6 2 1 9 1 6 2 6 2 1 9 1 6 2 6 2 1 9 1 6 2 6 2 1 10 1 6 2 6 2 1 1 1